hydra-web/src/js/modules/datatable.js

```diff
@@ -925,11 +925,17 @@ function(_, $, dt, Backbone){
                 checkbox.attr("class","checkbox checked-all");
                 checkbox.parent().data("state","all");
                 this.views.selectable.find("button.btn-hide-zero").removeClass("disabled");
+                this.views.selectable.find("button.btn-show-one").addClass("disabled");
```

Comment:
I'm not quite sure when the code in this if statement runs. Do you know what exactly the condition `if(_.isEqual(keys.length,this.collection.length))` is checking here?